solr/solrj-streaming/src/test/org/apache/solr/client/solrj/io/stream/eval/CoalesceEvaluatorTest.java

```diff
@@ -1,7 +1,7 @@
 /*
  * Licensed to the Apache Software Foundation (ASF) under one or more
  * contributor license agreements.  See the NOTICE file distributed with
- * this work for multitional information regarding copyright ownership.
+ * this work for additional information regarding copyright ownership.
```

Comment:
Interesting that the "file has correct license headers" checks didn't pick up on this.

Reply:
does it actually check the internal contents?    It looked like a set of cut n paste/ find/replace issues!